snaps/reference/rpc-api.md

```diff
@@ -562,6 +562,40 @@ console.log(entropy);
 
 <!--/tabs-->
 
+### snap_getLocale
+
+:::flaskOnly
+:::
+
+Gets the user's locale setting. You can use this method to localize text in your snap.
+
+#### Returns
+
+The user's locale setting.
```

Comment:
Should we provide detail on the specific locale format that's returned here? Is there a reference that can be included?

Reply:
@ziad-saab Do we expect the return format to be an ISO two-letter language code? https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes

Reply:
Hadn't thought about this. What is the format that MetaMask uses?

Reply:
OK, regardless of format, here's the locales: https://github.com/MetaMask/metamask-extension/blob/develop/app/_locales/index.json

What's the best way you suggest surfacing this information in the doc? Would be happy with a link to this file.